lib/reporter/reporters/minimalJson.js

```diff
@@ -0,0 +1,91 @@
+const lodashPick = require('lodash.pick');
+const path = require('path');
+const Utils = require('../../utils');
+const {Logger} = Utils;
+const BaseReporter = require('../base-reporter.js');
+
+class MinimalJsonReporter extends BaseReporter {
+
+  static isRerunFailedEnabled(argv = {}) {
+    return process.env.NIGHTWATCH_RERUN_FAILED === 'true' || argv['rerun-failed'];
+  }
+
+  static getRerunFailedFile(minimal_report_file_path) {
```

Comment:
@gravityvi I don't think keeping this logic in reporter makes sense. This is logic is related to running  of test cases, it should be in `test-source.js` itself.

Reply:
moved 👍